### ISSUE TITLE
Better support Adwaita Dark

### ISSUE THREAD

**Issue by Eonfge** [tchx84/Flatseal#7]
![Screenshot from 2020-01-31 22-59-23](https://user-images.githubusercontent.com/33983090/73577516-5f60c680-447d-11ea-9219-d1d05a5b56e3.png)

### Platform
Fedora 31
Flatpak 1.4.3

**Comment by tchx84** [comment]
Can you show me the Flatseal version as well?

**Comment by Eonfge** [comment]
```
Flatseal - A permissions manager for Flatpak

          ID: com.github.tchx84.Flatseal
         Ref: app/com.github.tchx84.Flatseal/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.1.2
     License: GPL-3.0-or-later
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 84,0 kB
     Runtime: org.gnome.Platform/x86_64/3.34
         Sdk: org.gnome.Sdk/x86_64/3.34

      Commit: 4859efdb68a5b505a0f5db516614190e711c310ef6e41893317150996050c97c
      Parent: 9b0f45ec74fb3bb715d7251de741917252b8956e75efeb40d9dfd211963c5e96
     Subject: Update to v1.1.2 (c716497d)
        Date: 2020-01-31 01:56:13 +0000
```

**Comment by tchx84** [comment]
Hmm, it seems my fix didn't work then. Ill take a better look tonight.

**Comment by tchx84** [comment]
Fixed it for real now, :man_facepalming:

**Comment by tchx84** [comment]
An update will be available in a few hours, the time it takes for Flathub to publish.

**Comment by tchx84** [comment]
Fixed in 1.2.2, just update from flathub.
